ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/answers/activityNumericAnswer.component.spec.ts

```diff
@@ -123,7 +112,6 @@ describe('ActivityNumericAnswer', () => {
         fixture.detectChanges();
         const inputElement: HTMLInputElement = fixture.debugElement.query(By.css('input')).nativeElement;
         component.numericField.patchValue(5);
-        component.onBlur('5');
         inputElement.dispatchEvent(new Event('blur'));
         fixture.detectChanges();
         expect(component.block.answer).toBe(5);
```

Comment:
Removed the onblur handler from component on template, so this does not do anything.